src/tools/clippy/tests/ui/cast_size.rs

```diff
@@ -1,6 +1,8 @@
 //@stderr-per-bitwidth
 //@no-rustfix
 
+#![feature(f128)]
+#![feature(f16)]
```

Comment:
Shouldn't be necessary, as long as the tests are commented out.

Reply:
Thanks, I removed these